packages/ui/src/design-system/select/select-root.component.tsx

```
@@ -0,0 +1,121 @@
+ * See: https://www.radix-ui.com/primitives/docs/components/select#portal
+ * @param required See: https://www.radix-ui.com/primitives/docs/components/select#root
+ * @param showArrow Render arrow icon next to the input value, when the Select is closed.
+ * @param value See: https://www.radix-ui.com/primitives/docs/components/select#root
+ * @param variant The style variant.
+ * @param triggerTestId The `data-testid` attribute, passed to the input trigger / root element.
+ */
+export const Root = ({
+  align = 'selected',
+  children,
+  disabled,
+  id,
+  name,
+  onChange,
+  defaultOpen = false,
+  placeholder,
+  portalContainer,
+  required,
+  showArrow = false,
+  value,
+  variant = 'plain',
+  triggerTestId,
+}: Readonly<SelectRootProps>): JSX.Element => {
+  const [isOpen, setIsOpen] = React.useState(defaultOpen);
+
+  return (
+    <Select.Root
+      name={name}
+      open={isOpen}
+      value={value}
+      required={required}
+      disabled={disabled}
+      onOpenChange={setIsOpen}
+      onValueChange={onChange}
+    >
+      <Select.Trigger
+        className={cx.trigger[variant]}
+        id={id}
+        data-testid={triggerTestId}
+      >
+        <Select.Value placeholder={placeholder} />
+        {showArrow && (
+          <Select.Icon asChild>
+            <ChevronDownIcon className={cx.triggerIcon} />
+          </Select.Icon>
+        )}
+      </Select.Trigger>
+      <Select.Portal container={portalContainer}>
+        <Select.Content
+          className={cx.content[variant]}
+          position={align === 'selected' ? 'item-aligned' : 'popper'}
+        >
+          <Select.Viewport>
+            {React.Children.map(children, child => {
+              if (isValidSelectRootChild(child)) {
+                return React.cloneElement(child, {
+                  variant,
+                });
+              }
+              console.error(
```

if we're saying that it 'must' be either an Item, or ItemRoot, we should throw here instead of just logging.